src/theme/typography.ts

```diff
@@ -84,6 +84,11 @@ export const typography: Typography = {
       color: colors.gray.darker,
       fontSize: fontSize.s
     },
+    bold: {
```

Comment:
is this present in the design system typography file?

Reply:
Got it added to Typography.